docs/DeveloperGuide.md

```diff
@@ -301,169 +364,208 @@ Priorities: High (Must-Have) - `* * *`, Medium (Nice-To-Have) - `* *`, Low (Not
 
 (For all use cases below, the **System** is `CareerSync` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Enter Internship Information From The Main Page**
+**Use Case: Enter Internship Entry From Main Window**
 
 **MSS**
-
-1. User accesses the main page. 
-2. User selects the option to enter internship information. 
-3. System prompts the user to input internship details such as company name, role title, description, etc. 
-4. User inputs the required internship details. 
-5. System validates the input data. 
-6. System saves the internship information. 
+1. User accesses the main page.
+2. User selects the option to add a new internship.
+3. System prompts the user to input internship details such as title, company, start/end dates, etc.
+4. User inputs valid internship details.
+5. System validates the input data.
+6. System saves the internship information.
 7. System displays a confirmation message indicating successful submission.
 
-    Use case ends.
+Use case ends.
+
+**Extensions** <br>
+1a. User is unable to access the main page. <br>
+1a1. System displays an error message. <br>
+5a. User inputs invalid internship details. <br>
+5a1. System displays an error message indicating the specific validation error(s). <br>
+6a. System fails to save the internship information. <br>
+6a1. System displays an error message and prompts the user to retry or cancel the submission.
+
+Use case ends.
+
+**Use Case: Edit Internship Entry From Main Window**
+
+**MSS**
+1. User accesses the main page.
+2. User selects the option to view all internship details.
```

Comment:
remove step 2 and 5
Combine steps 4 and 6 together
"User inputs the internship to be edited and the modifications"